cmd/gmailctl/cmd/io.go

```diff
@@ -56,6 +58,32 @@ func askOptions(prompt string, choices []string) int {
 	}
 }
 
+var colorFlag string
+
+func addColorFlag(cmd *cobra.Command) {
```

Comment:
Ah, one more thing: wouldn't it be easier to add this to the `rootCmd` as a global flag, just like `--config`? This way, we don't have to add it to every single command, and we also support aliases such as `alias gmailctl=gmailctl --color=never`.

Reply:
Done